### PR TITLE
Update to latest fedora-bootstrap (bootstrap5)

### PR DESCRIPTION
**Pull request by ryanlerch**

### CODE REVIEW

File: mirrormanager2/app.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def create_app(config=None):
     # Admin UI
     # Flask-Admin does not support having a single instance and multiple calls
     # to init_app() (it stores the app as an instance attribute)
-    ADMIN = Admin(template_mode="bootstrap3")
+    ADMIN = Admin(template_mode="bootstrap4")
```
Comment (ryanlerch) [inline mirrormanager2/app.py:108]: flask-admin doesnt have a bootstrap5 template mode, so we use the bootstrap4 one... it works ok enough for the admin pages

File: mirrormanager2/templates/fedora/admin/master.html [hunk 2]
```diff
@@ -1,94 +1,24 @@
 {% extends 'admin/base.html' %}
 {% import 'admin/lib.html' as lib with context %}
 {% import 'admin/static.html' as admin_static with context %}
+{% from "_macros.html" import footer, headincludes, nav, flashmessages, bottomincludes %}
```
Comment (ryanlerch) [inline mirrormanager2/templates/fedora/admin/master.html:4]: this template is why i used macros here -- cos the admin plugin basically needs its own master template, so used macros for the complex stuff here so we can re-use in the main master template

File: mirrormanager2/templates/fedora/host.html [hunk 3]
```diff
@@ -1,267 +1,246 @@
...
-          onclick="return confirm('You sure you want to delete this Peer ASN?');"
-          title="Delete Peer ASN">
-          <span class="icon icon-trash blue"></span>
-        </button>
-      </form>
-    {% else %}
-      {{ asn.asn }}
+    {% if not host.private %}
+    <div><strong>Last Crawled:</strong> {% if host.last_crawled %} {{ host.last_crawled
+      }} <a href="{{url_for('base.crawler_log', host_id=host.id)}}">[Log]</a> {% else %} - {% endif %} </div>
```
Comment (abompard) [inline mirrormanager2/templates/fedora/host.html:35]: Maybe "never" rather than "-" here?

**Review by abompard — CHANGES_REQUESTED** [review]
Looks great! Some remarks:
- all the flash messages for errors display in green, they should have the `error` category.
- you can drop the views and templates you no longer use, such as `host_category`.

File: mirrormanager2/templates/fedora/export_compliance.html [hunk 4]
```diff
@@ -0,0 +1,26 @@
+<div class="alert alert-light form-text">
```
[the remark this answers is not in the document]
Comment (ryanlerch) [reply mirrormanager2/templates/fedora/export_compliance.html:1]: whoops -- yeah didnt mean to add that file -- just going to keep this macro in _macros.html

removed this file

File: mirrormanager2/templates/fedora/maps_interactive.html [hunk 5]
```diff
@@ -1,4 +1,4 @@
-{% extends "master_noauth.html" %}
+{% extends "master" %}
```
[the remark this answers is not in the document]
Comment (ryanlerch) [reply mirrormanager2/templates/fedora/maps_interactive.html:1]: thanks for the catch! fixed.

File: mirrormanager2/templates/fedora/host.html (hunk 3 above)
Comment (ryanlerch) [reply mirrormanager2/templates/fedora/host.html:35]: yeah -- it was always '-' but never is better. fixed.

**Comment by ryanlerch** [comment]
> Looks great! Some remarks:
> 
> * all the flash messages for errors display in green, they should have the `error` category.
added a bunch of error categories to the flask.flash() calls
> * you can drop the views and templates you no longer use, such as `host_category`.
dropped the host_category page!

**Comment by ryanlerch** [comment]
Ready for re-review if you want

**Comment by abompard** [comment]
Looks great, anything you want to change before we merge it?

File: mirrormanager2/templates/fedora/site.html [hunk 6]
```diff
@@ -1,120 +1,90 @@
...
-  from participating in U.S. export transactions by any federal agency of the
-  U.S. government; or
-  (c) for use in connection with the design, development or production of
-  nuclear, chemical or biological weapons, or rocket systems, space launch
-  vehicles, or sounding rockets, or unmanned air vehicle systems.
-</p>
+<div class="mt-3 d-flex justify-content-between align-items-center">
+  <div>
+    <h2 class="mb-0"><span class="fa fa-globe"></span> {{ site.name }}</h2>
+  Created by <strong>{{ site.created_by }}</strong> at <strong>{{ site.created_at }}</strong>
```
Comment (abompard) [inline mirrormanager2/templates/fedora/site.html:12]: This looks a little bit weird:
image
Maybe do some formatting with the date? Nothing too fancy of course.

Comment (ryanlerch) [reply mirrormanager2/templates/fedora/site.html:12]: Yeah -- i started that, but involved adding a filter -- so will merge this, and fix in another PR

**Pull request merged**